src/components/HomePage/herosection/herosection.jsx

```diff
@@ -1,34 +1,44 @@
+import { useMediaQuery } from "react-responsive";
+
 import "../herosection/style.css";
 import placeHolder from "../../assets/placeholder1665.png";
 import bgCircleLM from "../../assets/bgCircleLM.svg";
 import bgCircleDM from "../../assets/bgCircleDM.svg";
 
-const herosection = ({ colorMode }) => {
+const Herosection = ({ colorMode }) => {
+	const isMobile = useMediaQuery({ query: `(max-width: 870px)` });
+
 	return (
 		<div id="heroSection">
-			<img
-				id="rightBgCircle"
-				src={colorMode ? bgCircleLM : bgCircleDM}
-				alt=""
-			/>
-			<img id="leftBgCircle" src={colorMode ? bgCircleLM : bgCircleDM} alt="" />
-			<div
-				id="heroContent"
-				style={colorMode ? { color: "#fff" } : { color: "#000" }}
-			>
+			{!isMobile && (
+				<>
+					<img
+						id="rightBgCircle"
+						src={colorMode ? bgCircleLM : bgCircleDM}
+						alt="Circle"
+					/>
+					<img
+						id="leftBgCircle"
+						src={colorMode ? bgCircleLM : bgCircleDM}
+						alt="Circle"
+					/>
```

Comment:
You have used the same images. check your `leftBgCircle` 

src for leftBgCircle-Linkify/src/components/HomePage/herosection/assets/dark-left.png

Reply:
Yes I did. Did you check the css files? This is some time back but the circles are the same so i just reused and rotated one...?